### ISSUE TITLE
Unable to get keyboard focus

### ISSUE THREAD

**Issue by pchandra2000** [annapogorelova/ng2-go-top-button#26]
Hi, 

I am using this control and it works great but there seems to be no way to get keyboard focus on the item.  i have tried putting tabindex='1' and adding a click handler to the <span> nothing has worked.

Here is my code:

  <go-top-button [animate]="true"
        [speed]="50"
        [acceleration]="2"
        [scrollDistance]="350"
        [classNames]="'go-top-scrolling'"
        tabindex='1'
        role='button'
        [styles]="{
          'border-radius': '20px 20px 20px 20px',
          'right': '15px',
          'bottom': '25px',
          'color': '#333',
          'border': '4px solid',
          'line-height': '20px'
        }">
        <span tabindex='1' class=\'fa fa-arrow-up\' ></span>
      </go-top-button>

Please help, if I cannot make this accessible I will have to take it off...

Regards,
Prashant.

**Comment by annapogorelova** [comment]
Thanks for letting me know about this issue, will check.

**Comment by annapogorelova** [comment]
@pchandra2000 Try the *v7.1.0* version of the package that I have just released. You can now set the `tabindex` attribute via `tabIndex` property of `<go-top-button>`, like this: 

```
  <go-top-button   [animate]="true"
                   [tabIndex]="1"
                   [speed]="50"
                   [acceleration]="2"
                   [scrollDistance]="100"
                   [styles]="{
                    'border-radius': '20px 20px 20px 20px',
                    'right': '5px',
                    'color': 'green',
                    'border': '5px solid',
                    'line-height': '20px'
                 }">
    Up
  </go-top-button>
```

**Comment by pchandra2000** [comment]
Thanks Anna,

I will try it shortly and let you know...

Regards,
Prashant.
